src/shared/types/index.js

```diff
@@ -152,6 +152,7 @@ export const incidentType = PropTypes.shape({
   status: PropTypes.shape({
     state: PropTypes.string,
     state_display: PropTypes.string,
+    send_email: PropTypes.bool,
```

Comment:
Do we know that `send_email` is the actual parameter that the API accepts/expects?

Reply:
Yes, I checked that with the backend-ers